cmd/external-plugins/cherrypicker/server.go

```diff
@@ -426,7 +426,8 @@ func (s *Server) handle(logger *logrus.Entry, requestor string, comment *github.
 	// Fetch the patch from GitHub
 	localPath, err := s.getPatch(org, repo, targetBranch, num)
 	if err != nil {
-		return fmt.Errorf("failed to get patch: %w", err)
+		logger.WithError(err).Errorf("Failed to get patch for %s/%s#%d", org, repo, num)
+		return s.createComment(logger, org, repo, num, comment, "Failed to get PR patch from GitHub. This PR will need to be manually cherrypicked.")
```

Comment:
Please embed the error (even if in a `<details>`) to short-circuit the support calls here - there might be other reasons for the error and we shouldn't need to go to server logs to see them.

Reply:
Added a details section to the comment.